c-boyle/future-hero
# Misc Changes (bug fixing and dialogue stuff)

## Changes:
- Make `TimedAudio` react appropriately to abrupt time remaining changes.
- Fix completion time screen bug where it wasn't resetting when the level was restarted.
- Add an NPC interaction where you can talk to one of the NPCs and they suggest that the cigarette smoking is dangerous. (Open to suggestions for a different NPC to have this interaction, or for different dialogue in general).
- Put question mark marker overtop NPCs that can be talked to (right now they only appear after a delay. The cashier question mark appears after 30 seconds, and the patron NPC question mark appears after a minute). The markers are removed once the cigarette is picked up, or if you do the interaction. The markers also don't stay on screen even when you're looking away, unlike the ruffian's leaving marker.
- Change the last line of the intro text.

## Screenshots:
Changed last line of intro text:
![image](https://user-images.githubusercontent.com/62191940/203250575-9134c8fe-8929-43ca-8473-32d585e1dad0.png)

Question mark above patron NPC:
![image](https://user-images.githubusercontent.com/62191940/203250833-f283956f-b4a8-4c89-9a05-6f9e806e06fc.png)

### How to Test:
Play the level, wait around for the markers to appear and make sure they disappear when it makes sense.
